README.md

```diff
@@ -1,6 +1,6 @@
 # Node.js
 
-[![dockeri.co](http://dockeri.co/image/_/node)](https://registry.hub.docker.com/_/node/)
+[![dockeri.co](https://dockerico.blankenship.io/image/node)](https://hub.docker.com/r/node)
```

Comment:
The image looks good, but the link change doesn't work
```suggestion
[![dockeri.co](https://dockerico.blankenship.io/image/node)](https://hub.docker.com/_/node)
```

Reply:
Thanks, fixed.